src/git/getDefaultRemote.ts

```diff
@@ -53,7 +53,7 @@ export function getDefaultRemote(cwdOrOptions: string | GetDefaultRemoteOptions)
   try {
     packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf8").trim());
   } catch (e) {
-    throw new Error(`Could not read "${packageJsonPath}"`);
+    logOrThrow(`Could not read "${packageJsonPath}"`);
```

Comment:
For this to work, you'll also need to the next line to do `const repository = packageJson?.repository`. Otherwise it will throw when attempting to access `packageJson.repository` since `packageJson` is undefined.

Reply:
`packageJson` is actually defined as `let packageJson: Partial<PackageInfo> = {};`, so it should be fine in this case, since it would be an empty object instead of undefined :)

Reply:
Ah okay, I missed that